services/venice-server/src/main/java/com/linkedin/venice/listener/StorageReadRequestsHandler.java

```diff
@@ -296,7 +296,7 @@ public void channelRead(ChannelHandlerContext context, Object message) throws Ex
           context.writeAndFlush(
               new HttpShortcutResponse("No storage exists for: " + e.getStoreName(), HttpResponseStatus.BAD_REQUEST));
         } catch (VeniceRequestEarlyTerminationException e) {
```

Comment:
Can you just update `VeniceRequestEarlyTerminationException` to use Netty's `HttpResponseStatus`?

Reply:
I don't follow. Do you mean remove the `HttpStatus` at [this line](https://github.com/linkedin/venice/blob/fabfe8912719b4a140f5416525c5991cb44a80b4/services/venice-server/src/main/java/com/linkedin/venice/listener/VeniceRequestEarlyTerminationException.java#L15) from `VeniceRequestEarlyTerminationException`?

Reply:
Updated.